### ISSUE TITLE
Add option to remove configuration only.

### ISSUE THREAD

**Issue by marcinpraczko** [erik/holepunch#7]
Hi, I think that would be really valuable to have option such as: '--remove-only'.
I've noticed that sometimes network sessions / tokens (depends of configuration) - can expired and then holepunch is not able to remove configuration, becuase is returing only error:

```bash
Skipping existing permission: {"ToPort": 443, "IpProtocol": "tcp", "IpRanges": ...
No changes to make.
```

So this means that my rule still exists - and I can't remove it. And running something like:
```bash
holepunch .... --remove-only
```
Could remove rule if exists.

**Comment by erik** [comment]
Thanks for the request!

This was originally an intentional decision on my part so that holepunch wouldn't clobber existing rules. I'm not sure if removing random rules falls into the scope of the project.

That said, I do think a flag similar to the one you suggested would be useful to have, maybe something like `--always-remove` or `--remove-existing` or something, which would clean up rules at exit, even if holepunch didn't originally create them.

What do you think? Would that work in your example?

**Comment by marcinpraczko** [comment]
Hi,

> This was originally an intentional decision on my part so that holepunch wouldn't clobber existing rules. I'm not sure if removing random rules falls into the scope of the project.

I totally agree that removing random rules is not right solution. My issue is that we are having tokens - which expires after some period of time. Holepunch is doing great work to add rules to SG's. However when token expires and even when I re-connect to create new token - holepunch is not able remove old rules any more. So I have to stop holepunch and when re-run getting error described above (that roles already exists).

I would suggest any solution which will allow remove rules if match some pattern, for example (just suggestion).

```bash
# Cerate rules
holepunch ..... (all options here)
# ---> hole punch save JSON to local folder with data

# Delete rules (when token expired, etc)
holepunch .... --remove-existing
# ---> hole punch will compare rules with saved JSON - if match - will remove, if not match will complain with error
```

That was just suggestion - so no random removal - only work easier with tokens (which are used during federation). Maybe there are others soutions like - re-read token, etc? Not sure - now when we are having profiles - maybe holepunch is able work better with tokens. Not sure - first I need remove rules which were created :) 

Hope that helps. I hope that I explained where my issue is so far.
Summary: I am not asking for removing 'random' rules - only those which were created with holepunch - or syntax match during creation and removal.

**Comment by erik** [comment]
Aha! 

Thank you for the additional context, that really helps in understanding the issue. I guess I've never left a rule in place for long enough to run into the token issue that you described.

So in addition to the flag, I'll fix the bug where the AWS token expires. That should hopefully be a pretty simple change.

**Comment by erik** [comment]
Hi @marcinpraczko,

I just released `holepunch==0.5.0` which supports the changes I described. It should fix removal after the token expires, and has a new flag, `--remove-existing` which will ensure that rules will be cleared out at exit regardless of whether holepunch created them or not.

Please let me know if this doesn't solve your issue or you run into any bugs!

**Comment by marcinpraczko** [comment]
HI @erik 

First of all - big thank you for so fast and great work. I will test this options soon and let you know. From what I can see in code, it should work really well. Very, very well done.

**Comment by marcinpraczko** [comment]
Hi @erik 

I've just tested this option and seems be working fine. Only one thing which I noticed that this was not working on Python2 any more and had to start using Python3. Which was not big deal. Except this - everything is working really nice. Thank you.

In terms of tokens - we will see this later today, rules are up and running and will check behaviour of holepunch after while whether token is re-read as expected.
